example/lib/main.dart

```diff
@@ -81,7 +81,15 @@ class FontAwesomeGalleryHomeState extends State<FontAwesomeGalleryHome> {
               child: Column(
                 mainAxisAlignment: MainAxisAlignment.center,
                 children: <Widget>[
-                  Hero(tag: icon, child: FaIcon(icon.iconData)),
+                  Hero(
+                      tag: icon,
+                      child: icon.iconData != null
+                          ? FaIcon(icon.iconData)
+                          : FaDuotoneIcon(
+                              icon.duotoneIconData as IconDataDuotone,
+                              primaryColor: Colors.black,
+                              secondaryColor: Colors.blueGrey[400],
+                            )),
```

Comment:
This uses the same approach as #98. The problem is that it introduces way more complexity into the example app than I like. That's why #98 is still open.

Reply:
Unfortunately, I don't see a way to do this that doesn't add complexity. Because of the need to have a different Icon type (`FaIcon` vs `FaDuotoneIcon`), there has to be some sort of switch somewhere to identify which to use.

I tried a few different approaches, and this was the simplest of them all.  I also don't think this is that complex - it's just a basic check, if the icon has iconData (which indicates it's not duotone), then use it.  Otherwise, use the duotone data.

Reply:
What if I did something like the following:

```dart
Hero(
                      tag: icon,
                      child: icon.iconData != null
                          ? FaIcon(icon.iconData)
                          : FaIcon(null)),
                         // : FaDuotoneIcon(
                         //     icon.duotoneIconData as IconDataDuotone,
                         //     primaryColor: Colors.black,
                         //     secondaryColor: Colors.blueGrey[400],
                         //   )),
```

Then, in the readme under the pro section, added instructions to comment out the null line and uncomment the duotone lines.  That would allow me to also stop exposing the `FaDuotoneIcon` API, which simplifies things how you wanted.